### PR TITLE
LR-382: Update LICENSE url

### PR DESCRIPTION
**Pull request by caohhung**
---

Supported PR commands:

| Command                | Description            |
| ---------------------- | ---------------------- |
| `ok to test`           | Re-run standard checks |
| `extended test`        | BackstopJS tests       |
| `extended check sonar` | SonarQube tests        |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/2685/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4315/ : SUCCESS in 15m 40s

File: README.md [hunk 1]
```diff
@@ -44,6 +44,6 @@ In case the pull brings in new projects or large bulk of changes, it is safer (a
 
 (C) 2017-2020 GoodData Corporation
 
-This repository is under the a GoodData commercial license available in the [LICENSE](LICENSE) file because it contains a
+This repository is under the a GoodData commercial license available in the [LICENSE](https://github.com/gooddata/gooddata-ui-sdk/blob/release/LICENSE) file because it contains a
```
Comment (no23reason) [inline README.md:47]: The correct link should probably be: https://github.com/gooddata/gooddata-ui-sdk/blob/master/LICENSE

**Comment by no23reason** [comment]
We should also update the READMEs of

- sdk-ui-charts
- sdk-ui-all
- sdk-ui-ext
- sdk-ui-tests
- sdk-examples

They will probably have a similar problem

**Comment by no23reason** [comment]
Fixed in #1095

**Pull request closed without merging**